tests/sentry/integrations/slack/service/test_slack_service.py

```diff
@@ -92,59 +92,64 @@ def setUp(self) -> None:
 
     def test_none_group(self):
         self.activity.update(group=None)
-        self.service._logger = mock.MagicMock()
-        self.service.notify_all_threads_for_activity(activity=self.activity)
-        self.service._logger.info.assert_called_with(
-            "no group associated on the activity, nothing to do",
-            extra={"activity_id": self.activity.id},
-        )
+
+        with mock.patch.object(self.service, "_logger") as mock_logger:
```

Comment:
The fact that we're patching it in tests makes `_logger` effectively part of the module's public contract. IMHO that would justify removing the leading underscore.